WeatherManComputations.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime
+
+import WeatherManFileHandle
+
+
+class Computations:
```

Comment:
Let's rename the class to a Meaningful name:
e.g: WeatherRegister, WeatherList etc.